turbinia/task_manager.py

```diff
@@ -436,6 +436,18 @@ def process_result(self, task_result):
     Returns:
       TurbiniaJob|None: The Job for the processed task, else None
     """
+    if task_result.successful is None:
```

Comment:
I think we want to measure if this happens often, can you add a metric for this? This file already has some metrics defined so should be easy to add.
https://github.com/google/turbinia/blob/72c918a59ff9b89691cd11715b6aef109140d241/turbinia/task_manager.py#L57

Reply:
Good idea.  Done.